components/table/TableData.jsx

```diff
@@ -41,11 +42,17 @@ const TableData = ({
                     key={uniqueId()}
                     className={`usda-table__row${oddClass}`}>
                     {row.map((data, j) => (
-                        <td
-                            key={uniqueId()}
-                            className={`usda-table__cell${columns[j]?.right ? ' usda-table__cell_right' : ''}`}>
-                            {data}
-                        </td>
+                        columns[j]?.header ?
+                            <TableHeader
+                                key={uniqueId()}
+                                className={data?.title ? 'nested-header' : 'empty'}
```

Comment:
can we use a different class name for when this evaluates to truthy?

Reply:
Do we even need to pass `className` here if no special styles are applied? The TableHeader component doesnt make it a required prop.